fast64_internal/f3d/f3d_gbi.py

```diff
@@ -2001,7 +2001,7 @@ def vertexScrollTemplate(
 
 
 class GfxFormatter:
-    def __init__(self, scrollMethod: ScrollMethod, texArrayBitSize: int, seg2virtFuncName: Union[str, None]):
+    def __init__(self, scrollMethod: ScrollMethod, texArrayBitSize: int = 8, seg2virtFuncName: Union[str, None]):
```

Comment:
How does this solve mk64's endianess issues exactly? this will still get set to 64 bit or 8 bit by all the dl exporters, i think we could implement an mk64 dl exporter after this pr and or add an enum/int in the general f3d exporter for the bit size

Reply:
![image](https://github.com/user-attachments/assets/2eb7d0bc-8b4e-4d2f-a88b-1334c2b85a94)
u16

Reply:
u8
![image](https://github.com/user-attachments/assets/23e183fd-9f13-4f3e-aa32-eaf6843d6a40)

Reply:
![image](https://github.com/user-attachments/assets/ab17f740-7e75-4c96-8be8-7d4e06fe04dc)
Perhaps this is where I need to change 64 to 8?

Reply:
I tested the above and that works. Should I change that to 8?

Reply:
Or leave this PR as is